code/modules/mob/living/simple_animal/hostile/syndicate.dm

```diff
@@ -86,12 +85,17 @@
 	name = "Syndicate Commando"
 	loot = list(/obj/effect/gibspawner/human)
 	speed = 1
+	var/obj/effect/light_emitter/red_energy_sword/sord
 
 /mob/living/simple_animal/hostile/syndicate/melee/space/Initialize()
 	. = ..()
 	sord = new(src)
 	set_light(4)

```

Comment:
light's value should be a var on parent and then you can remove this `set_light` aswell